app/templates/forum.html

```diff
@@ -21,7 +21,7 @@ <h2>Last Post</h2>
     {% for thread in threads %}
     <div class="post-title">
         <a href="{{ url_for('forum_post', groupID=groupID, threadID=thread.id) }} ">
-            <h1>{{thread.title}}</h1>
+            <h1>{{post.title}}</h1>
```

Comment:
```suggestion
            <h1>{{thread.title}}</h1>
```
Only threads have titles.

Reply:
Yea, my bad. Meant to make the change for the thread content, i.e post.content but was late night and groggy. Will make the changes